monitor/FlowMonitor.js

```diff
@@ -1022,14 +1022,20 @@ module.exports = class FlowMonitor {
       "e.dest.ports": this.getRemotePorts(flowObj),
       "p.from": intelObj.from,
       "p.intf.id": flowObj.intf,
-      "p.tag.ids": flowObj.tags
+      "p.tag.ids": flowObj.tags,
+      "p.originIP": intelObj.originIP
     };
 
     this.updateURLPart(alarmPayload, flowObj);
 
     let alarm = new Alarm.IntelAlarm(flowObj.ts, deviceIP, severity, alarmPayload);
-
-
+    if (intelObj.isOriginIPIP) {
+      alarm['p.alarm.becauseof'] = 'ip';
+    } else if(intelObj.isOriginIPAPattern){
+      alarm['p.alarm.becauseof'] = 'subdomain';
```

Comment:
use the actual domain instead of the *subdomain* or *domain* :)

because there may be more domain layers. e.g. docs.aws.amazon.com

* docs.aws.amazon.com
* aws.amazon.com
* amazon.com